core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -841,7 +841,6 @@ public void setAttribute() throws Exception {
     Assert.assertEquals(1, fileInfo.getTtl());
 
     // Set ttl for a directory, raise IllegalArgumentException.
```

Comment:
I think 843 and the next line should be removed too since they are meant to test something not limited any more